src/znchelper.c

```diff
@@ -98,6 +98,7 @@ static void znc_write_chat(
 	PurpleConversation *conv, const char *who, const char *message,
 	PurpleMessageFlags flags, time_t mtime
 ) {
+	PurpleConnection *gc = purple_conversation_get_gc(conv);
```

Comment:
Also please don't mix up declarations and code and make sure to check the return values.

Reply:
I'll fix that right now.

Reply:
commit added to fix this